Jenkinsfile.deploy

```diff
@@ -131,7 +131,7 @@ pipeline {
             }
             steps {
                 script {
-                    sh "make run-tests timeout=360 load_factor=3 concurrent=6 image_cache_mode=always smoke=${params.SMOKE_TESTS_ONLY} ${getBDDParameters()}"
+                    sh "make run-tests timeout=360 load_factor=3 concurrent=3 image_cache_mode=always smoke=${params.SMOKE_TESTS_ONLY} ${getBDDParameters()}"
```

Comment:
Set back to 3 for now. Anyway with scenario concurrency, it should already be a bit faster